src/test/resources/Homework17.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="Homework17">
+    <parameter name="baseURL" value="https://bbb.testpro.io" />
+
+    <test name="Example Test" preserve-order="false">
+        <classes>
+            <class name="Homework17">
+                <methods>
+                    <include name="playSong" />
+                </methods>
+            </class>
+        </classes>
+    </test>
+</suite>
+
+<!--<class name="RegistrationTests"/>-->
+<!--<class name="ChangeUsernameTests"/>-->
```

Comment:
To run the test, here is the fix:

```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >

<suite name="Homework17">
    <parameter name="baseURL" value="https://bbb.testpro.io" />
    <test name="Homework17" preserve-order="false">
        <classes>
            <class name="Homework17">
                <methods>
                    <include name="addSongToPlaylist" />
                </methods>
            </class>
        </classes>
    </test>
</suite>
```

![image](https://user-images.githubusercontent.com/104761482/200163335-b8c82dec-e377-40f3-a04b-4980ba48815e.png)